File: java/returnNegative.java
// makes any number a negative

public class returnNegative {
  public static int makeNegative(final int x) {
    return x > 0 ? -1 * x : x;
  }

  public static void main(String[] args) {
    System.out.println(makeNegative(1)); // -1
    System.out.println(makeNegative(0)); // 0
    System.out.println(makeNegative(-13)); // -13
  }
}

File: c#/ReturnNegative.cs
// convert all input integers to make them all negative

using System;


public class ReturnNegative
{

	public static int MakeNegative(int num)
	{
		return num > 0 ? -num : num;
	}

	public static void Main(string[] args)
	{
		Console.WriteLine(MakeNegative(3)); // -3
		Console.WriteLine(MakeNegative(-3)); // -3
	}
}
